.travis.yml

```diff
@@ -29,15 +29,41 @@ env:
     - SEQ_PG_PW=sequelize_test
     - SEQ_PG_HOST=127.0.0.1
     - SEQ_PG_PORT=8998
+    # oracle info
+    - NODE_ORACLEDB_HOST: localhost
+
+before_install:
+  - |
+    if [ $ORACLE_VER ]; then 
+      mkdir -p /opt/oracle && 
+      cd /opt/oracle && 
+      wget https://s3.amazonaws.com/sequelize/instantclient-basic-linux.x64-12.1.0.2.0.zip &&
```

Comment:
I have no idea what the new version adds; I can't find any changelog. But for my tests and dev, I'm with 12.1

Reply:
Instant Client is a packing of client libraries covering a number of products, so the doc is in a number of places.  For node-oracledb one change is the improvement in "High Availability" due to session pool enhancements.  I'd recommend using 12.2 client.  It will connect to older DBs too, of course.

Reply:
I don't have access to this server, can someone that has access can update this dep ?

Reply:
Yes

Reply:
https://s3.amazonaws.com/sequelize/instantclient-basic-linux.x64-12.2.0.1.0.zip
https://s3.amazonaws.com/sequelize/instantclient-sdk-linux.x64-12.2.0.1.0.zip

Reply:
I've changed the paths, thanks :)